consultation_analyser/consultations/jinja2/consultations/consultations/show.html

```diff
@@ -17,43 +17,43 @@
 
   {%- from "govuk_frontend_jinja/components/breadcrumbs/macro.html" import govukBreadcrumbs -%}
 
+  <style>
+    .govuk-main-wrapper {
+      background: white;
+    }
+  </style>
+
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
-
-      <iai-question-tiles></iai-question-tiles>
-        <script>
-          const questionTilesEl = document.getElementsByTagName("iai-question-tiles")[0];
-
-          questionTilesEl.consultationName = "{{ consultation.title }}"
-          questionTilesEl.questions = [
-            {% for question_dict in all_questions_list %}
-              {
-                id: "{{ question_dict.question.id }}",
-                title: "Question {{ question_dict.question.number }}",
-                body: `{{ question_dict.question.text }}`,
-                url: "{{ url('question_responses', kwargs={'consultation_slug': consultation.slug, 'question_slug': question_dict.question.slug}) }}",
-                responses: {
-                  {% if question_dict.free_text_question_part %}
-                    count: {{ question_dict.free_text_count }}
-                  {% endif %}
-                },
-                multiResponses:
-                  {% if question_dict.single_option_question_part %}
-                    {{ question_dict.single_option_counts | safe }}
-                  {% elif question_dict.multiple_option_question_part %}
-                    {{ question_dict.multiple_option_counts | safe }}
-                  {% else %}
-                    {}
-                  {% endif %}
-              },
-            {% endfor %}
-          ]
-        </script>
+      <iai-question-overview-page></iai-question-overview-page>
+      <script>
+        const questionOverviewPage = document.getElementsByTagName("iai-question-overview-page")[0];
+
+        questionOverviewPage.consultationName = "{{ consultation.title }}"
+        questionOverviewPage.questions = [
+          {% for question_dict in all_questions_list %}
+            {
+              id: "{{ question_dict.question.id }}",
+              title: "{{ question_dict.question.text }}",
+              url: "{{ url('question_responses', kwargs={'consultation_slug': consultation.slug, 'question_slug': question_dict.question.slug}) }}",
+              numResponses:
+                {% if question_dict.has_free_text %}
+                  {{ question_dict.free_text_count }}
```

Comment:
I think this is always displaying as 0 as I don't think `has_free_text` is one of the keys of the dictionary (don't know why we're not getting an error?).

Basically, I think:
```
   {% if question_dict.has_free_text %}
          {{ question_dict.free_text_count }}
    {% else %}
      0
    {% endif %}
```

should be replaced with ```{{ question_dict.free_text_count }}``` (which will exist in the dictionary whether or not we have multiple choice parts or not).

Once we start counting multi choice, this count might need to be amended.

Reply:
Now done!